gitlab/types.py

```diff
@@ -63,8 +63,8 @@ def get(self) -> Any:
     def set_from_cli(self, cli_value: Any) -> None:
         self._value = cli_value
 
-    def get_for_api(self) -> Any:
-        return self._value
+    def get_for_api(self, *, key: str) -> Tuple[str, Any]:
+        return (key, self._value)
```

Comment:
I've been thinking even earlier with the FileAttribute that it's kind of a roundabout way that we have to supply the key manually when we actually define it along with the type. We could probably refactor this later to just have the key on instantiation or so, and leave the method signatures as before. Then we also don't need to have conversations about keyword arguments :rofl: